src/main/scala/org/ergoplatform/nodeView/mempool/ErgoMemPoolReader.scala

```diff
@@ -14,11 +14,16 @@ trait ErgoMemPoolReader extends MempoolReader[ErgoTransaction] {
 
   def getAll: Seq[ErgoTransaction]
 
-  /** Returns all transactions resided in pool sorted by priority */
+  /** Returns all transactions resided in pool sorted by priority
+    */
   def getAllPrioritized: Seq[ErgoTransaction]
 
   def take(limit: Int): Iterable[ErgoTransaction]
 
   def modifierById(modifierId: ModifierId): Option[ErgoTransaction]
 
+  /** Returns a list of randomly selected transactions
+    */
+  def digest(size: Int): Seq[ErgoTransaction]
```

Comment:
@oskin1 write @param description and you will see that `size` is not natural name.